radio/util/tts_de.py

```diff
@@ -18,63 +18,63 @@
                             (u"MilliAmpere", "mamp0"),
                             (u"Knoten", "knot0"),
                             (u"Meter pro sekunde", "mps0"),
-                            (u"Fuesse pro sekunde", "fps0"),
-                            (u"kilometer pro stunde", "kph0"),
+                            (u"Fuess pro sekunde", "fps0"),
+                            (u"Kilometer pro stunde", "kph0"),
                             (u"Meilen pro Stunde", "mph0"),
                             (u"Meter", "meter0"),
                             (u"Fuesse", "foot0"),
-                            (u"Grad", "celsius0"),
+                            (u"Grad Celsius", "celsius0"),
                             (u"Grad Fahrenheit", "fahr0"),
                             (u"Prozent", "percent0"),
-                            (u"mMilliampere pro stunde", "mamps0"),
+                            (u"Milliampere pro stunde", "mamps0"),
                             (u"Watt", "watt0"),
                             (u"Milli Watt", "mwatt0"),
                             (u"D B", "db0"),
-                            (u"R P M", "rpm0"),
+                            (u"Drehzahl", "rpm0"),
                             (u"g", "g0"),
-                            (u"degree", "degree0"),
+                            (u"Grad", "degree0"),
                             (u"radians", "rad0"),
                             (u"Milliliter", "ml0"),
                             (u"Unze", "founce0"),
                             (u"Uhr", "hour0"),
                             (u"Minuten", "minute0"),
-                            (u"Secunden", "second0"),
+                            (u"Sekunden", "second0"),
```

Comment:
Well, it will be there when you add it (I think previous translater did put "Uhr"